app/assets/stylesheets/base/_base.scss

```diff
@@ -1,5 +1,36 @@
+@import 'utilities/variables';
 
 body {
-	background-color: $--primary-color-dark;
-  	font-size: 18px;
+  background-color: $--primary-color-dark;
+  font-size: 18px;
 }
+
+a {
+  text-decoration:none;
+}
+
+.container{
```

Comment:
add a space between the class name and your curly brace

Reply:
done